README.md

```diff
@@ -1,2 +1,81 @@
 # django-ilmoitin
-Django app for sending notifications
+
+[![Latest PyPI version](https://badge.fury.io/py/django-ilmoitin.svg)](https://pypi.python.org/pypi/django-ilmoitin)
+[![Python versions](https://img.shields.io/pypi/pyversions/django-ilmoitin.svg)](https://pypi.python.org/pypi/django-ilmoitin)
+
+A templated Django messaging library
+
+# Installation
+
+1. `pip install django-ilmoitin`
+
+2. Add `django_ilmoitin` to `INSTALLED_APPS`.
+
+# Usage
+
+1. `django-ilmoitin` uses [`django-mailer`](https://github.com/pinax/django-mailer) to send emails, so you need to configure the `MAILER_EMAIL_BACKEND` setting to let [`django-mailer`](https://github.com/pinax/django-mailer) know, how to actually send the mail:
+```python
+MAILER_EMAIL_BACKEND = "your.actual.EmailBackend"
+```
+
+2. Create a `notifications.py` file in django app and register your notification types:
+
+```python
+from django_ilmoitin.registry import notifications
+
+notifications.register("event_created", "Event created")
+```
+
+3. Create a `dummy_context.py` file in django app and add dummy context data:
+
+```python
+from django_ilmoitin.dummy_context import dummy_context
+
+from .models import MyModel
+
+model = MyModel(foo="bar")
+
+dummy_context.content.update({
```

Comment:
you renamed it to `dummy_context.context`. I had old code, when I gave you my example 🙂 update this line, please.